spamhandling.py

```diff
@@ -17,11 +17,8 @@ def should_whitelist_prevent_alert(user_url, reasons):
     if not is_whitelisted:
         return False
     reasons_copy = list(set(reasons))
-    reasons_copy_copy = list(reasons_copy)
-    for reason in reasons_copy:
-        if "username" in reason:
-            reasons_copy_copy.remove(reason)
-    return len(reasons_copy_copy) == 0
+    reasons_comparison = [r for r in list(reasons_copy) if "username" not in r]
+    return len(reasons_comparison) == 0
```

Comment:
Wouldn't it make sense to use [`any`](http://stackoverflow.com/q/19389490/359284#19389957) here?

```python
return any(r for r in set(reasons) if "username" not in r)
```

Also, you shouldn't need to call `list` if you are making it a `set` ahead of time.